LondonBritishLibrary/orient/BLorient485.xml

```diff
@@ -121,7 +246,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <origDate notBefore="1500" notAfter="1525" evidence="lettering">
                             </origDate>
                         </origin>
-                        <provenance>The manuscript had very probably belonged to a church in the <placeName ref="LOC3577Gondar"/> area, 
+                        <provenance>
+                            Several features indicate a possible Beta ʾƎsrāʾel origin of this manuscript (see  <bibl><ptr target="bm:DegeMueller2020BE"/><citedRange unit="page">18</citedRange></bibl>
+                            on the doxology on <locus target="#1r"/>, and note also the absence of trinitarian formulas integrated into the main text, and 
+                            the absence of a cross or other decoration on the quarter cover). It is probable that the manuscript changed owners repeatedly, before
+                            it came into the possession of <placeName ref="INS0101MadhaneAlam"/>.
```

Comment:
A bit shaky. I assume, Old Testament books do not receive a trinitarian formula in CHristian manuscripts generally, unless the contrary is proven. :) Quarter cover must not be decorated with cross also at Christians. The only thing is the formula ይትባረክ፡, which is an added one, even though by the hand not too different from the main one. In my opinion, we could think only that the ms might have been in possession of Beta Esrael for some time.

Reply:
This is exactly what the article argues.

Reply:
Denis argues against the BE origin; he says it may have been in BE possession for some time - after its origin.

Reply:
So does Sophia. Now I see that my use of the word "origin" here was not very exact, I had not meant the production. I will change the formulation. Thank you!

Reply:
replace it with "connection" )